src/PlanTrip/Trip.tsx

```diff
@@ -0,0 +1,141 @@
+import { Feature, Leg, TripPattern } from "../types";
+import { ActionPanel, Action, Color, List } from "@raycast/api";
+import { fetchTrip } from "../api";
+import { useCallback, useEffect, useState } from "react";
+import {
+  getTransportIcon,
+  formatAsClock,
+  formatTimeDifferenceAsClock,
+  formatMetersToHuman,
+} from "../utils";
+import Accessory = List.Item.Accessory;
+
+const buildAccessories = (trip: TripPattern, isDetailsVisible: boolean): Accessory[] => {
+  if (isDetailsVisible) {
+    return [
+      {
+        tag: {
+          value: formatTimeDifferenceAsClock(trip.expectedStartTime, trip.expectedEndTime),
+          color: Color.Blue,
+        },
+      },
+    ];
+  }
+  const accessories = trip.legs
```

Comment:
Setting the type explicitly here should remove the need for `// @ts-expect-error` below.

```suggestion
  const accessories: Accessory[] = trip.legs
```